src/commands/deployWorkspaceProject/getDefaultValues/getWorkspaceProjectPaths.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, UserCancelledError, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { Uri, WorkspaceFolder, commands, workspace } from "vscode";
+import { DOCKERFILE_GLOB_PATTERN, browseItem } from "../../../constants";
+import { localize } from "../../../utils/localize";
+import { getRootWorkspaceFolder } from "../../../utils/workspaceUtils";
+
+export async function getWorkspaceProjectPaths(context: IActionContext): Promise<{ rootFolder: WorkspaceFolder, dockerfilePath: string }> {
+    const prompt: string = localize('selectRootWorkspace', 'Select a project with a Dockerfile');
+    const rootFolder: WorkspaceFolder | undefined = await getRootWorkspaceFolder(prompt);
+
+    if (!rootFolder) {
+        await context.ui.showQuickPick([browseItem], { placeHolder: prompt });
+        await commands.executeCommand('vscode.openFolder');
+
+        // Silently throw an exception to exit the command while VS Code reloads the new workspace
+        throw new UserCancelledError();
+    }
+
+    // Check if chosen workspace has a Dockerfile at its root
+    const dockerfileUris: Uri[] = await workspace.findFiles(DOCKERFILE_GLOB_PATTERN);
+    if (!dockerfileUris.length) {
+        throw new Error(localize('noDockerfileError', 'Unable to locate a Dockerfile in your project\'s root.'));
+    } else if (dockerfileUris.length > 1) {
+        throw new Error(localize('multipleDockerfileError', 'Multiple Dockerfiles found. Unable to determine the correct Dockerfile to use in your project\'s root.'));
```

Comment:
I feel like we could just show the user a quick pick of dockerfiles so that they can select which one that they want. Maybe after we merge this, but I think that's kind of a must since some projects have test dockerfiles, prod dockerfiles, etc.

Forcing them to move/delete the files seems like a crummy UX.

Reply:
That's a good call, I agree

Reply:
Added todo